APCAppCore/APCAppCore/Startup/APCTabBarViewController.m

```diff
@@ -67,8 +66,6 @@ - (void)passcodeViewControllerDidSucceed:(APCPasscodeViewController *)viewContro
     self.isPasscodeShowing = NO;
     self.showPasscodeScreen = NO;
     [viewController dismissViewControllerAnimated:YES completion:nil];
-    [[NSUserDefaults standardUserDefaults] setObject: [NSDate date] forKey:kLastUsedTimeKey];
```

Comment:
Capturing the timestamp of when the user was authenticate by the app is an approximation for the *Last Used* timestamp in the event the user deletes the app from the *Recently Used* list. As such, these two lines cannot be deleted unless there's another mechanism for capturing the *Last Used* timestamp.